data/campaigns/Heir_To_The_Throne/scenarios/23_Test_of_the_Clans.cfg

```diff
@@ -27,7 +27,7 @@
 				if not cfg.silent then
 					local splash = _("Still $units_to_slay clan member to defeat!", "Still $units_to_slay clan members to defeat!", n)
 					wesnoth.wml_actions.print{
-						text = splash:vformat{units_to_slay = n},
+						text =_  splash:vformat{units_to_slay = n},
```

Comment:
This is dead wrong, as this is Lua code rather than WML. It will result in an error at runtime.

@Elvish-Hunter Is it easy to prevent wmllint from touching embedded Lua code?

Reply:
reverted in ad68ef0

Reply:
> This is dead wrong, as this is Lua code rather than WML. It will result in an error at runtime.
> 
> @Elvish-Hunter Is it easy to prevent wmllint from touching embedded Lua code?

@hexagonrecursion has a fix that does that in #5470